OpenSim/Simulation/Model/Station.h

```diff
@@ -31,53 +31,64 @@
 #include "OpenSim/Simulation/Model/PhysicalFrame.h"
 
 namespace OpenSim {
+    
```

Comment:
It's standard for us to not indent the contents of the namespace's scope. Could you remove this indent? If your editor did this automatically, it might also have an option to prevent indenting the namespace's scope.